files/default/rules/hopsfs.rules.yml

```diff
@@ -46,4 +46,11 @@ groups:
       summary: "Out of HopsFS storage"
       description: "HopsFS Storage (< 20% left)\n  VALUE = {{ $value }}"
 
-
+  - alert: HopsFSNamenodeDown
+    expr: absent(namenode_FSNamesystemState_capacity_used) > 0
+    for: 5m
+    labels:
+      type: system-alert
+    annotations:
+      summary: "HopsFS Namenode is unresponsive"
+      description: "HopsFS Namenode is not reporting metrics"
```

Comment:
The problem with this type of alerts is that if you have HA, you don't get an alert if 1 Namenode is down, as you might have another namenode being available. 

So I think we either need to make the alert more granular (i.e. 1 namenode is not responding) or we should change the summary saying: "The entire file system is down".